### DISCUSSION TITLE
Real purpose of OpenVDB and GLFW in PicoGK

### DISCUSSION THREAD

**Discussion by hariharan382** [leap71/PicoGK#45] [General]
HI, your work greatly inspired me, but, I am not getting real use of the usage of OpenVDB(voxel based) data structure, why was it used in pickGK, and what challenges does the openvdb solve for computational engineering?

And I found GLFW, to be a frontend for Windows and Mac, not sure if this conclusion is true or not, Thanks for such a wonderful package

**Comment by LinKayser** [comment]
PicoGK uses OpenVDB as geometry engine (Voxels, etc.). And it uses GLFW to show a user interface (the viewer).
Lin
